plugins/tekton/src/utils/pods-data.ts

```diff
@@ -0,0 +1,116 @@
+import { V1Pod } from '@kubernetes/client-node';
```

Comment:
Thinking about this further after the first look, I wonder if this should go in a test file or more clearly indicate that this is test stuff that shouldn't get built into the actual plugin code.  I'm sure that's not happening now, but you never know, that can be changed down the line accidentally.

Reply:
moved this test data file to `__fixtures__` folder to indicate this is test related file.